src/BQ24195.cpp

```diff
@@ -1195,6 +1202,68 @@ bool PMICClass::isBatteryInOverVoltage() {
     return 0;
 }
 
+
+/*******************************************************************************
+ * Function Name  : setFastChargeTimerSetting
+ * Description    : Sets the charging safety timer
+ * Input          : safety timer duration in hours (rounds up)
+ * Return         : 0 on Error, 1 on Success
+ *******************************************************************************/
+bool PMICClass::setFastChargeTimerSetting(float hours) {
+
+    int DATA = readRegister(CHARGE_TIMER_CONTROL_REGISTER);
```

Comment:
```suggestion
    int const DATA = readRegister(CHARGE_TIMER_CONTROL_REGISTER);
```
Same here. Also I personally would be writing `DATA` lowercase, as all upper-case are reserve for constants usually.